### DISCUSSION TITLE
List of flags

### DISCUSSION THREAD

**Discussion by revolvedai** [smy20011/dreambooth-gui#94] [Q&A]
Hello, is there anywhere to find a list of all the flags and what they do? I'm kind of piecing some of it together from auto1111, but am curious what flags are out there. Would be great to add the functionality from the colabs to this gui via flags for example! Thanks for the great platform.

**Comment by smy20011** [comment]
All the supported flags are here:
https://github.com/ShivamShrirao/diffusers/blob/main/examples/dreambooth/train_dreambooth.py#L35
